### ISSUE TITLE
Add ELMAH (NuGet package) for logging Exceptions

### ISSUE THREAD

**Issue by florisrobbemont** [imazen/keyhub#30]
Add Elmah package to the project and configure it to only allow access to users from the admin role. Also change the current logging implementation to send Fatal, Error and Warning messages to Elmah as well.

**Comment by florisrobbemont** [comment]
Added ELMAH logging to sysadmin role. You can reach it at: http://siteurl/elmah
This contains all errros on the website. We should optionally change the current nlog logger to log to Elmah as well

**Comment by lilith** [comment]
Appfail.net is also an option, if it saves dev time; I've used it in the past.

**Comment by dragan** [comment]
@nathanaeljones Do you want to do this ourselves or would you rather use a service like [Appfail.net](http://appfail.net/) or [raygun.io](http://raygun.io/). I haven't used raygun, but I've heard very good things.

**Comment by lilith** [comment]
Do you know if Raygun also measures response times? It would be good to be able to measure latency as well - I think NewRelic does that too.

**Comment by dragan** [comment]
No, it's strictly for tracking errors that happen during runtime. We would still need to use NewRelic in conjunction.
